docs/pages/color_usage.js

```diff
@@ -0,0 +1,270 @@
+          **$color-text-success** - Use this token as text color to indicate success.
+          **$color-text-warning** - Use this token as text color to indicate a warning or caution.
+          **$color-text-error** - Use this token as text color to indicate an error.
+          `}
+        >
+          <Flex gap={4} wrap>
+            <ColorTile description="Success" textColor="white" fullTokenName="color-text-success" />
+            <ColorTile description="Warning" textColor="white" fullTokenName="color-text-warning" />
+            <ColorTile description="Error" textColor="white" fullTokenName="color-text-error" />
+          </Flex>
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Inverse text color"
+          description={`
+          Colors used on top of inverted backgrounds. Use the tokens:
+          **$color-text-inverse** - Use this token for text layered on top of inverted backgrounds.
+          `}
+        >
+          <ColorTile description="Inverse" fullTokenName="color-text-inverse" />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection
+        name="Iconography color"
+        description={`
+      The icon colors available indicate status and match the messaging text colors for consistency. For icon colors purposes, use the tokens:
+
+      **$color-text-icon-default** - Use this token as the default color for icons.
+      **$color-text-icon-subtle** - Use this token as the secondary color for icons.
+      **$color-text-icon-success** - Use this token for success icons.
+      **$color-text-icon-warning** - Use this token for warning icons.
+      **$color-text-icon-error** - Use this token for error icons.
+      **$color-text-icon-inverse** - Use this token for icons paired with inverted backgrounds.
+      `}
+      >
+        <Flex gap={4} direction="column">
+          <Flex gap={4} wrap>
+            <ColorTile
+              description="Default"
+              textColor="white"
+              fullTokenName="color-text-icon-default"
+            />
+            <ColorTile
+              description="Subtle"
+              textColor="white"
+              fullTokenName="color-text-icon-subtle"
+            />
+          </Flex>
+          <Flex gap={4} wrap>
+            <ColorTile
+              description="Success"
+              textColor="white"
+              fullTokenName="color-text-icon-success"
+            />
+            <ColorTile
+              description="Warning"
+              textColor="white"
+              fullTokenName="color-text-icon-warning"
+            />
+          </Flex>
+          <Flex gap={4} wrap>
+            <ColorTile
+              description="Error"
+              textColor="white"
+              fullTokenName="color-text-icon-error"
+            />
+            <ColorTile description="Inverse" fullTokenName="color-text-icon-inverse" />
```

Comment:
this needs dark text
![Screen Shot 2022-02-16 at 11 36 33 AM](https://user-images.githubusercontent.com/12059539/154342333-54367900-685b-4115-9990-b85140b5c249.png)